tests/vmi_hook_sidecar_test.go

```diff
@@ -63,14 +63,16 @@ var _ = Describe("[sig-compute]HookSidecars", func() {
 	})
 
 	Describe("[rfe_id:2667][crit:medium][vendor:cnv-qe@redhat.com][level:component] VMI definition", func() {
-		getVMIPod := func(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, error) {
+		getVMIPod := func(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, bool, error) {
 			podSelector := tests.UnfinishedVMIPodSelector(vmi)
 			vmiPods, err := virtClient.CoreV1().Pods(vmi.GetNamespace()).List(context.Background(), podSelector)
 
-			if err != nil || len(vmiPods.Items) != 1 {
-				return nil, fmt.Errorf("could not retrieve the VMI pod: %v", err)
+			if err != nil {
+				return nil, false, fmt.Errorf("could not retrieve the VMI pod: %v", err)
+			} else if len(vmiPods.Items) == 0 {
```

Comment:
I could express the `exists` boolean with something like `len(vmiPods.Items) != 0`, is that what you mean? Otherwise I think I need it, like  I wrote it right now.

Reply:
Sorry for not being clear. I was just referring to `else` keyword. But this is definitely nit :)

Reply:
I can change it, no general issues with it but I always like giving the first case priority and as such enforcing an order. it is harder to change the order by accident if an `else if` is there.

Reply:
Interesting thought. I am ok with both versions.